server/monitor/src/main/java/org/apache/accumulo/monitor/servlets/trace/Basic.java

```diff
@@ -51,13 +51,14 @@
 
   public static String getStringParameter(HttpServletRequest req, String name,
       String defaultValue) {
-    String result = req.getParameter(name).replaceAll("[^A-Za-z]", "");
+    final String result = req.getParameter(name);
     if (result == null) {
       return defaultValue;
     }
     return result;
   }
 
+
```

Comment:
This extra whitespace will probably get deleted when the project is built, due to the formatter plugin. Please run the formatter with `mvn package -DskipTests` and update the PR with any changes.

Reply:
Will do.

Reply:
Done.